style.css

```diff
@@ -171,171 +205,181 @@ margin-left: 50px;
-    padding-top: 50px;
-    text-align: center;
-    font: 24px Mensch, sans-serif;
-    color: #e8e5d2;
-  }
-
-  .shape_left img {
-    bottom: 14px;
-    left: -54px;
-    position: absolute;
-    float: left;
-  }
-  .shape_right img {
-    position: absolute;
-    float: right;
-    bottom: 14px;
-    right: -54px;
-  }  
-  .info-posts a {
-    display: block;
-    
-    text-align: left;
-    font: 14px Tahoma, sans-serif;
-    color: #e8e5d2;
-  }
-  .info-posts {
-    display: block;
-    padding-top: 7px;
-    text-align: left;
-    font: 14px Tahoma, sans-serif;
-    color: #e8e5d2;
-  }
-  .info-posts span {
-    text-align: left;
-    font: 14px Tahoma, sans-serif;
-    color: #878473;
-  }
- 
- .posts {
+.title_footer {
+  text-transform: uppercase;
+  padding-bottom: 0;
+  padding-top: 50px;
+  text-align: center;
+  font: 24px Mensch, sans-serif;
+  color: #e8e5d2;
+}
+.brd {
+  margin-top: 9px;
+  border-bottom: 0.1px solid #605f5b;
+  width: 220px;
+}
+.shape_left img {
+  bottom: 14px;
+  left: -54px;
+  position: absolute;
+  float: left;
+}
+.shape_right img {
+  position: absolute;
+  float: right;
+  bottom: 14px;
+  right: -54px;
+}  
+.info_posts a {
+  display: block;
+  text-align: left;
+  font: 14px Tahoma, sans-serif;
+  color: #e8e5d2;
+}
+.info_posts {
+  display: block;
+  padding-top: 7px;
+  text-align: left;
+  font: 14px Tahoma, sans-serif;
+  color: #e8e5d2;
+}
+.info_posts span {
+  text-align: left;
+  font: 14px Tahoma, sans-serif;
+  color: #878473;
+}
+.posts {
   margin-left: 17px;
   float: left;
   width: 220px;
-  }
-  .archives {
+}
+.archives {
   margin-right: 17px;
   margin-left: 17px;
   float: left;
   width: 220px; 
-  }
-  .dummy {
-    float: left;
-    width: 220px;
-  }
-  .search  {
-    margin-left: 17px;
-    float: left;
-    width: 220px;
-  }
-  .info_line {
-    background: #61615d;
-    height: 0.1px;
-    width: 220px;
-  }
-  .end_footer {
-    margin: 0 auto;
-    width: 980px;
-    clear: both;
-    background: #4d4d4d;
-    position: relative;
-  }
-  .arrow {
+}
+.dummy {
+  float: left;
+  width: 220px;
+}
+.search  {
+  margin-left: 17px;
+  float: left;
+  width: 220px;
+}
+.end_footer {
+  margin: 0 auto;
+  width: 980px;
+  clear: both;
+  background: #4d4d4d;
+  position: relative;
+}
+.arrow {
   position: relative;
   left: 490px;
   bottom: 17px;
-  }
-  .by {
-    padding: 0 0 25px;
-    text-align: center;
-    font: 14px Tahoma, sans-serif;
-    color: #fff;
-  }
-  .by span {
-    text-align: center;
-    font: 14px Tahoma, sans-serif;
-    color: #fdd600;
-  }
-
+}
+.by {
+  padding: 0 0 25px;
+  text-align: center;
+  font: 14px Tahoma, sans-serif;
+  color: #fff;
+}
+.by span {
+  text-align: center;
+  font: 14px Tahoma, sans-serif;
+  color: #fdd600;
+}
+.search input {
+  background: #efecd9;
+  margin-top: 10px;
+  margin-left: -20px;
+  width: 220px;
+  height: 40px;
+}
+.button_search {
+  position: absolute;
+  top: 1767.7px;
+  left: 1090px;
```

Comment:
absolute.. with huge `left` value.. It should be related to parent div.. not to whole page or whatever global